mdrss.go

```diff
@@ -58,16 +59,27 @@ func parseCommand(command string, config mdrss.Config) error {
 }
 
 func main() {
-  if len(os.Args) != 2 {
-    mdrss.Error.Println("mdrss <<update, ls, conf >>")
-    return
+  var config_path string
+  flag.StringVar(&config_path, "config", "~/.mdrss/config.json", "path to config.json")
+  flag.Parse()
+
+  var cmd string
+  switch len(os.Args) {
+    case 2:
+      cmd = os.Args[1]
+    case 4:
+      cmd = os.Args[3]
```

Comment:
This is something I'm not a huge fan of; because it's guessing whether the config flag comes before or after the command.